html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head><title>html forms</title></head>
+  <body>
+    <h1>  Coding Form! </h1>
+    <div>
+      <form action="http://email.com/email/php">
+
+        <label>Email: </label>
```

Comment:
This `label` still needs to be configured to match with its form control element.